src/components/Installment/InstallmentHeader.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { connect } from "react-redux";
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from "reactstrap";
+import AddInstallmentForm from "./AddInstallmentForm";
+import { theme } from "../../config";
+import { StyledTd } from "../../styles/InstallmentStyles";
+
+function InstallmentHeader(props) {
+  const [modal, setModal] = useState(false);
+
+  const toggleModal = e => {
+    e.preventDefault();
+    setModal(!modal);
+  };
+
+  return (
+    <HeaderContainer>
+      <div className="header">
+        <h2>Installments</h2>
+
+        <Button className="add-installment" onClick={toggleModal}>New</Button>
+      </div>
+      <table>
+        <thead>
+          <tr>
+            <StyledTd className="amountPaid">Amount Paid</StyledTd>
+            <StyledTd className="mode">Mode</StyledTd>
+            <StyledTd className="datePaid">Date Paid</StyledTd>
+            <StyledTd className="officer">Officer</StyledTd>
+            <StyledTd className="actions-head">
+              Actions
+            </StyledTd>
+          </tr>
+        </thead>
+      </table>
+
+      <Modal isOpen={modal} toggle={toggleModal}>
+        <AddInstallmentForm toggleModal={toggleModal} />
+      </Modal>
+    </HeaderContainer>
+  );
+}
+
+export default connect(
+  null,
+  {
+    // Actions
+  }
+)(InstallmentHeader);
+
+const HeaderContainer = styled.div`
+  width: 100%;
+  position: -webkit-sticky;
+  position: sticky;
+  top: 0;
```

Comment:
Shout out to @chasegarsee for the sticky, that was a great idea. pushes component from good to great.